.github/workflows/backend.yml

```diff
@@ -0,0 +1,89 @@
+name: Backend CI/CD pipeline
+
+on:
+  push:
+    branches: [ main ]
+    paths: [ backend/** ]
+  pull_request:
+    paths: [ backend/** ]
```

Comment:
Does this mean we deploy even when we create a pull request?

Reply:
No only when we push check the of statement in the deploy step